usb-mount.sh

```diff
@@ -61,7 +62,7 @@ do_mount()
 
     # File system type specific mount options
     if [[ ${ID_FS_TYPE} == "vfat" ]]; then
-        OPTS+=",users,gid=100,umask=000,shortname=mixed,utf8=1,flush"
+        OPTS+=",users,user,gid=65534,uid=65534,umask=000,shortname=mixed,utf8=1,flush"
```

Comment:
This may not be required. When `guest ok = yes` is passed in smb.conf, the default guest account(`nobody` here) is invoked for the share. Now, if you are not able to write to the share, it's probably because of conflicting ACLs. I see that you have have also set `create mask = 0660`, which denies any sort of access for 'others'(`rw-rw----`) on newly created files. If the owner and group of the files is not the user linked with the 'guest account', access is denied. Right ACLs must be set or(and) changing the default guest account user helps as well.